extension/js/common/assert.ts

```diff
@@ -35,7 +35,7 @@ export class Assert {
 
   public static abortAndRenderErrOnUnprotectedKey = async (acctEmail?: string, tabId?: string) => {
     if (acctEmail) {
-      const primaryKi = await KeyStore.getFirstOptional(acctEmail);
+      const primaryKi = await KeyStore.getFirstRequired(acctEmail);
```

Comment:
Here you switched to required, but you can see below that we are testing `&& primaryKi &&` for truthiness. That means it was expected that this key may be optional in this particular method. With this in mind, you should also scrutinize other parts.

Reply:
@tomholub Yes, I've evaluated all usages of `getFirstRequired` vs `getFirstOptional`. I had mistakenly used `getFirstRequired` but then I fixed it in the next commit.

Reply:
Can you please check the latest commit? You'll see I fixed it.